CONTRIBUTING.md

```diff
@@ -65,6 +65,8 @@ In general, `factory/.env` master should contain the latest versions we official
 
 ![](https://github.com/cypress-io/cypress/assets/1271364/85507060-acc3-48b5-bc16-4160c4620e1e)
 
+If you need to test that the image works with Cypress, you can follow this [gist](https://gist.github.com/AtofStryker/da2e94e8535cffeebf676b6399f96b47) if on a MacOS machine which might prove helpful when debugging image dependencies.
```

Comment:
Should we move the `gist` to a more official location? Or add the contents in here directly?

Reply:
I think gists can only be created by users, but we can add the contents directly here if we think thats the right direction. I'll go ahead and do that since i'm fine with either

Reply:
updated in [a3aab3e](https://github.com/cypress-io/cypress-docker-images/pull/1041/commits/a3aab3e20cd75900200897a1bed32a64d73419c2)